bootstrap_setup.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+export BOOTSTRAP_VERSION="v5.3.2"
```

Comment:
could move the `v` prefix out of the variable then use 5.3.2 in the remaining bits of the script that haven't used the version variable